.github/workflows/package-set.yml

```diff
@@ -0,0 +1,29 @@
+name: "package-set"
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  verify:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+        path: base-checkout
+    - uses: actions/checkout@v2
+      with:
+        repository: dfinity/vessel-package-set
+        path: vessel-package-set
+    - name: "install vessel"
+      run: |
+        mkdir -p /home/runner/bin
+        echo "/home/runner/bin" >> $GITHUB_PATH
+        wget --output-document /home/runner/bin/vessel https://github.com/kritzcreek/vessel/releases/download/v0.6.0/vessel-linux64
```

Comment:
It might be generally good practice to put such variable things like version numbers into an `env` section at the top, to make it easier to spot what one has to change.

Reply:
Something I did over here is to actually pull the "installing" bit out into its own action. That way we could change the relevant versions all in one place: https://github.com/dfinity/vessel-package-set/blob/81331bc430180ef0ce9c942b2b598eefc8a29667/.github/actions/install-tools/action.yml